server/src/test/java/io/spine/server/aggregate/AggregateStorageLifecycleFlagsHandlingTest.java

```diff
@@ -63,8 +62,8 @@ void writeEntityStatusOfAggregate() {
                                               .build();
         storage.writeLifecycleFlags(id, status);
         Optional<LifecycleFlags> readStatus = storage.readLifecycleFlags(id);
-        assertTrue(readStatus.isPresent());
-        assertEquals(status, readStatus.get());
+        Truth8.assertThat(readStatus)
```

Comment:
Please use static imports. The compiler should be able to tell `assertThat(Optional)` from `assertThat(Comparable)`.